src/function.py

```diff
@@ -453,6 +461,11 @@ def _package_log_payload(data):
         }
     ]
 
+    nr_tags_str = _get_newrelic_tags()
+    if nr_tags_str:
```

Comment:
Why not put this logic in the `_get_newrelic_tags` function?

Reply:
Thanks for the feedback. I have made this change.